src/operator/l2_normalization-inl.h

```diff
@@ -108,8 +108,8 @@ class L2NormalizationOp : public Operator {
       Tensor<xpu, 2> norm = out_data[l2_normalization::kNorm]
         .get_with_shape<xpu, 2, real_t>(norm_shape, s);
       norm = reduce_with_axis<red::sum, false>(F<mxnet::op::mshadow_op::square>(data), 2);
-      norm = F<mxnet::op::mshadow_op::square_root>(norm);
-      out = data / broadcast_with_axis(norm + param_.eps, 1, dshape[2]);
+      norm = F<mxnet::op::mshadow_op::square_root>(norm + param_.eps);
+      out = data / broadcast_with_axis(norm, 1, dshape[2]);
```

Comment:
The gradient of the original L2Normalization is wrong if we consider the eps.
Previously it's \frac{x}{\sqrt{\sum x^2} + eps}, and the gradient is quite complicated.
Now we change it to \frac{x}{\sqrt{\sum x^2 + eps}} and the gradient can be calculated as in the backward.